lib/readline.js

```diff
@@ -929,17 +932,27 @@ function emitKeypressEvents(stream, iface) {
   stream[ESCAPE_DECODER] = emitKeys(stream);
   stream[ESCAPE_DECODER].next();
 
+  const escapeCodeTimeout = () => stream[ESCAPE_DECODER].next('');
+
   function onData(b) {
     if (stream.listenerCount('keypress') > 0) {
       var r = stream[KEYPRESS_DECODER].write(b);
       if (r) {
+        if (timeoutRef) {
+          timeoutRef = clearTimeout(timeoutRef);
```

Comment:
This does not return anything. See https://github.com/nodejs/node/blob/ede98a77677afcc5e75a126043a11a720f7ae28b/lib/timers.js -- you can use just use `clearTimeout(timeoutRef)` without the `if` or re-assigning.